sw/host/opentitanlib/src/io/gpio.rs

```diff
@@ -92,4 +106,12 @@ pub trait GpioPin {
         }
         Ok(())
     }
+
+    /// Set up edge trigger detection on this pin, transport will buffer the list internally.
```

Comment:
Curious about style. Could we provide a default implementation here that is not supported since most of the transports do not support it, and it is more of an advanced feature. Thoughts?

Reply:
I had the same thought, and think it would be worth not having to touch 5+ files just to add "unimplemented" error code, which could sit in a default implementation.  This would be the first such methods in the io traits, though.  Chris, what are your thoughts?